src/media/js/addon/components/addon.js

```diff
@@ -97,6 +109,8 @@ export class Addon extends React.Component {
           <dd>{this.props.version}</dd>
         </dl>
 
+        {this.renderDescription()}
```

Comment:
i like this style pattern for simple conditionals in JSX:

```js
{this.props.description &&
  <div>
    // ...Description stuff.
  </div>
}
```

Otherwise, this can probably just be put in the definition list with all the other fields.